Code/Bench/inchi.cpp

```diff
@@ -0,0 +1,66 @@
+#include <catch2/catch_all.hpp>
+#include <string>
+
+#include "bench_common.hpp"
+
+#include <GraphMol/ROMol.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <INCHI-API/inchi.h>
+
+using namespace RDKit;
+
+TEST_CASE("MolToInchi", "[inchi]") {
+  auto samples = bench_common::load_samples();
+  BENCHMARK("MolToInchi") {
+    std::vector<std::string> inchis;
+    for (auto &mol : samples) {
+      ExtraInchiReturnValues rv;
+      inchis.push_back(MolToInchi(mol, rv));
+    }
+    return inchis;
+  };
+}
+
+TEST_CASE("MolToInchiKey", "[inchi]") {
```

Comment:
If you're benchmarking `MolToInchi()` and `InchiToInchiKey()`, this one is redundant.